openedx/features/course_experience/tests/views/test_course_home.py

```diff
@@ -89,7 +110,7 @@ def test_queries(self):
         course_home_url(self.course)
 
         # Fetch the view and verify the query counts
-        with self.assertNumQueries(51):
-            with check_mongo_calls(5):
+        with self.assertNumQueries(46):
```

Comment:
I think this is because that final test sets the unified_waffle_view to be turned on, which passes into this test and requires less queries - I'm surprised it would reduce the total a full five though.